pkg/minikube/download/image.go

```diff
@@ -168,17 +169,40 @@ func ImageToCache(img string) error {
 	}
 }
 
+func parseImage(img string) (*name.Tag, name.Reference, error) {
+	digest, err := name.NewDigest(img)
+	if err == nil {
+		tag := digest.Tag()
+		return &tag, digest, nil
+	}
+
+	_, ok := err.(*name.ErrBadName)
+	if !ok {
+		return nil, nil, errors.Wrap(err, "new ref")
```

Comment:
change this to a better wrap
```
errors.Wrap(err, "bad name")
```

Reply:
It's not a bad name. It's bad image reference

Reply:
then change to "bad ref" so it is different than the next one which is also "new ref"

Reply:
But for callers it's the same error - `img` contains no valid image reference. 
 "new ref" is a terrible error message, actually, but it's already in the code

Reply:
changed